docs/source/user-guide/concepts/channels.rst

```diff
@@ -23,8 +23,8 @@ For details, see how to :ref:`modify your channel lists <config-channels>`.
 
 We use conda-forge as an example channel.
 `Conda-forge <https://conda-forge.org/>`_ is a community channel
-made up of thousands of contributors. Conda-forge itself is analogous to PyPI
```

Comment:
The comparison to PyPI is important.  Does it need more detail?  What would it take to make it a better part of this section?

Reply:
It can stay in. We have a reference to conda-forge and PyPI on the main landing page but it is probably important for it to be elaborated upon here.

Reply:
I put the language back in there. Our readers probably understand the importance of that statement better than I did.